src/org/zaproxy/zap/extension/httppanel/view/syntaxhighlight/lexers/WwwFormTokenMaker.java

```diff
@@ -481,13 +481,8 @@ public void yypushback(int number)  {
     int zzMarkedPosL;
     int zzEndReadL = zzEndRead;
     char [] zzBufferL = zzBuffer;
-    char [] zzCMapL = ZZ_CMAP;
-
-    int [] zzTransL = ZZ_TRANS;
-    int [] zzRowMapL = ZZ_ROWMAP;
-    int [] zzAttrL = ZZ_ATTRIBUTE;
 
-    while (true) {
+      while (true) {

```

Comment:
This whitespace change should be reverted? (In the github interface at least this makes the while aligned with code it contains.)

Reply:
I'd suggest reverting all changes to this file, they will be lost once we regenerate it.

Reply:
@kingthorin, @thc202, all changes reverted in this file.